src/Rest/RestApiBrowser.php

```diff
@@ -150,11 +161,69 @@ public function addRequestHeader($name, $value)
      */
     private function removeRequestHeader($headerName)
     {
+        $headerName = strtolower($headerName);
         if (array_key_exists($headerName, $this->requestHeaders)) {
             unset($this->requestHeaders[$headerName]);
         }
     }
 
+    /**
+     * @param string $name
+     * @param string|stream $contents
+     */
+    private function addRequestMultipart($name, $contents) {
+        $multipart = [];
+        $multipart['name'] = $name;
+        $multipart['contents'] = $contents;
+        $this->requestMultiparts[] = $multipart;
+    }
+
+    /**
+     * @param string $name
+     * @param string $path
+     */
+    public function addFileToRequest($name, $path) {
+        $file = [];
+        $file['name'] = $name;
+        $file['path'] = $path;
+        $this->requestFiles[] = $file;
+    }
+
+    /**
+     * @param string $body
+     * @return \GuzzleHttp\Psr7\MultipartStream
+     */
+    private function getBodyAsMultipartFormData($body) {
+        if ($body) {
+            $this->transformRequestJsonBodyIntoMultipart($body);
+        }
+        $this->transformRequestFilesIntoMultipart();
+
+        $boundary = sha1(uniqid('', true));
+        $this->setRequestHeader('Content-Type', 'multipart/form-data; boundary='.$boundary);
+        return new \GuzzleHttp\Psr7\MultipartStream($this->requestMultiparts, $boundary);
+    }
+
+    /**
+     * @param string|array $body
```

Comment:
I see that the $body param of sendRequest() method may be an array type.
https://github.com/ubirak/rest-api-behat-extension/blob/master/src/Rest/RestApiBrowser.php#L94

For sure? Is there a case of using it as array in the project?
If so I should handle this here?